### PR TITLE
Russian translation of "We Accept Monero"/"How To Buy Monero"/"index"

### PR DESCRIPTION
**Pull request by v1docq47**
@lh1008 @erciccione
Another portion of translations into Russian ;)

Translated:
- \translations\ru\archive\how_to_buy_ru.md
- \translations\ru\archive\we_accept_monero_ru.md
- \translations\ru\website-sections\index_ru.md

Ping to review:
@LvMsterfild @monerorus

### CODE REVIEW

**Review by LvMsterfild — APPROVED** [review]

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### [Monero Outreach](https://www.monerooutreach.org/)
+
+### [Telergam](https://t.me/joinchat/DM8-zUXYPy2QPEkH5hW3FA)
+
+### [Twitter](https://twitter.com/xmroutreach)
+
+Официальный веб-сайт Monero
+
+[getmonero.org](https://www.getmonero.org/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:45]: https://www.getmonero.org/ru/index.html

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Видео](https://www.monerooutreach.org/videos/)
+
+- [Галерея](https://www.monerooutreach.org/gallery/)
+
+- [Лучшие практики Monero](https://www.monerooutreach.org/monero_best_practices/)
+
+- [Ресурсы](https://www.monerooutreach.org/resources.php)
+
+- [Ресурсы для инвесторов](https://www.monerooutreach.org/investors/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:33]: https://www.monerooutreach.org/ресурсы-инвестора/

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Истории](https://www.monerooutreach.org/stories/)
+
+- [Видео](https://www.monerooutreach.org/videos/)
+
+- [Галерея](https://www.monerooutreach.org/gallery/)
+
+- [Лучшие практики Monero](https://www.monerooutreach.org/monero_best_practices/)
+
+- [Ресурсы](https://www.monerooutreach.org/resources.php)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:31]: https://www.monerooutreach.org/pесурсы-monero.html

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Домой](https://www.monerooutreach.org/)
+
+- [Истории](https://www.monerooutreach.org/stories/)
+
+- [Видео](https://www.monerooutreach.org/videos/)
+
+- [Галерея](https://www.monerooutreach.org/gallery/)
+
+- [Лучшие практики Monero](https://www.monerooutreach.org/monero_best_practices/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:29]: https://www.monerooutreach.org/monero-Лучший-практика/

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Истории](https://www.monerooutreach.org/stories/)
+
+- [Видео](https://www.monerooutreach.org/videos/)
+
+- [Галерея](https://www.monerooutreach.org/gallery/)
+
+- [Ресурсы](https://www.monerooutreach.org/resources.php)
+
+### _Архив историй_ - [Смотреть все](https://www.monerooutreach.org/stories/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:13]: https://www.monerooutreach.org/Рассказы/

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+### _Архив видео_ - [Смотреть все](https://www.monerooutreach.org/videos/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:15]: https://www.monerooutreach.org/Видео/

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+### _Архив галереи_ - [Смотреть все](https://www.monerooutreach.org/gallery/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:17]: https://www.monerooutreach.org/галерея/

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+## Нижний колонтитул
+
+- [Домой](https://www.monerooutreach.org/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:21]: The same from the above

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Истории](https://www.monerooutreach.org/stories/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:23]: The same from the above

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Видео](https://www.monerooutreach.org/videos/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:25]: The same from the above

File: monero-outreach-docs/translations/ru/website-sections/index_ru.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Галерея](https://www.monerooutreach.org/gallery/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/index_ru.md:27]: The same from the above

**Comment by v1docq47** [comment]
@lh1008 thanks for the corrections ;)

**Review by lh1008 — APPROVED** [review]
Thank you @v1docq47 

:)

**Comment by lh1008** [comment]
Pinging @erciccione

**Pull request merged**